ip-messaging/rest/channels/create-channels/create-channels.5.x.rb

```diff
@@ -0,0 +1,15 @@
+require 'http'
+require 'twilio-ruby'
+
+# Initialize the client
+account_sid = 'ACCOUNT_SID'
+auth_token = 'AUTH_TOKEN'
+ip_messaging_client = Twilio::REST::IpMessagingClient.new(
```

Comment:
TL;TR

This one won't work. The _next-gen_ version does not have an `Twilio::REST::IpMessagingClient` -- Keep in mind that the next version only exposes a `Twilio::REST::Client`.

Besides that, there is no `get` method exposed in the public interface of the next-gen Twilio SDK.

You can see that the hierarchy of classes goes like this:

```
ListResource
  InstanceContext
    InstanceResource
```
Where:

| Base class | Defined methods |
|---|---|
| `ListResource` | `list`, `create`|
| `InstanceContext` | `fetch` |
| `ListResource` | `delete`, `update`|

e.g.:

```ruby
client = Twilio::REST::Client(...)

client.calls #=> CallList < ListResource
client.account.calls('some random SID') #=> CallContext < InstanceContext
client.account.calls('some random SID').fetch #=> CallInstance < InstanceResource
```